assets/js/main.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var weatherAppModule = (function () {
+    const displayWeatherData = (data) => {
+        const cityName = document.getElementById('citytitle');
+        const content1 = document.getElementById('content1');
+        const content2 = document.getElementById('content2');
+
+        const text = ` The temperature is ${Math.floor(data.main.temp)} Kelvin , a humidity of ${Math.floor(data.main.humidity)} % 
+        and a pressure of ${Math.floor(data.main.pressure)} hPa`;
+
+        const text2 = document.createElement('p');
+
+        cityName.innerText = data.name;
+        content1.innerText = text;
+        text2.innerText = data.weather[0].description;
+        content2.innerText = '';
+
+        const img = document.createElement('img');
+        img.setAttribute('src', `http://openweathermap.org/img/w/${data.weather[0].icon}.png`);
+
+        content2.append(text2, img);
+
+    };
+    const getCityWeather = (city) => {
+        fetch(`https://api.openweathermap.org/data/2.5/weather?q=${city}&appid=99a238e2a89a4068350516ce6a439bec`, { mode: 'cors' })
```

Comment:
let make this asynchronous using async/await or promises. we don't know what the result will be returned.
 and abstract the request.